docs/scripts/make_model_zoo_docs.py

```diff
@@ -227,6 +227,14 @@
     dataset.apply_model(model, label_field="predictions")
 
     session = fo.launch_app(dataset)
+
```

Comment:
@manushreegangwar be careful about adding empty lines in this file: they are literally added to the example code in the published docs 💀 

In this case, can you please remove the empty line after the two `launch_app()` lines affected by this diff?